src/Impostor.Server/Net/Inner/Objects/InnerShipStatus.cs

```diff
@@ -56,31 +56,13 @@ public override async ValueTask DeserializeAsync(IClientPlayer sender, IClientPl
                 return;
             }
 
-            if (initialState)
+            while (reader.Position < reader.Length)
             {
-                // TODO: (_systems[SystemTypes.Doors] as DoorsSystemType).SetDoors();
-                foreach (var systemType in SystemTypeHelpers.AllTypes)
+                IMessageReader msgReader = reader.ReadMessage();
```

Comment:
Please don't use `msg` it looks weird, also vars, rest of the code uses just `var message`

Reply:
```suggestion
                var message = reader.ReadMessage();
```